src/eureka/S4_generate_lightcurves/s4_genLC.py

```diff
@@ -463,6 +494,8 @@ def genlc(eventlabel, ecf_path=None, s3_meta=None, input_meta=None):
                 # Plot each spectroscopic light curve
                 if meta.isplots_S4 >= 3:
```

Comment:
Wow, I'm surprised this bug hasn't been spotted sooner.  This should be `if meta.isplots_S4 >= 1:`